sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java

```diff
@@ -548,6 +554,10 @@ private void updateFacility() {
 		final DistrictReferenceDto district;
 		final CommunityReferenceDto community;
 
+		if (UserRole.getJurisdictionLevel(UserProvider.getCurrent().getUserRoles()) == JurisdictionLevel.HEALTH_FACILITY) {
```

Comment:
please move this if statement the fist line in in the method.

The field declarations belong to the if statement below, it's not a good idea to insert something not related between the declaration and value assignment.